wiki/Tournaments/ASC/2/en.md

```diff
@@ -0,0 +1,208 @@
+| Streamer | ::{ flag=VN }:: [- Fubukiii](https://osu.ppy.sh/users/9931217), ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=NZ }:: [Penguin Bread](https://osu.ppy.sh/users/5091293), ::{ flag=CN }:: [Rush\_FTK](https://osu.ppy.sh/users/3046856) |
+| Commentator | ::{ flag=US }:: [aero-](https://osu.ppy.sh/users/26690747), ::{ flag=GB }:: [epic man 2](https://osu.ppy.sh/users/14566000), ::{ flag=CA }:: [ERA Sunny](https://osu.ppy.sh/users/16468962), ::{ flag=PH }:: [Itawachi](https://osu.ppy.sh/users/12929973), ::{ flag=MY }:: [IvanOfficial](https://osu.ppy.sh/users/12818249), ::{ flag=PH }:: [Lazereed](https://osu.ppy.sh/users/12894120), ::{ flag=CN }:: [Rush\_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=MY }:: [Unicow](https://osu.ppy.sh/users/9444174), ::{ flag=HK }:: [YonGin](https://osu.ppy.sh/users/7109317) |
+| Designer | ::{ flag=ID }:: [Aikiyo Fuuka](https://osu.ppy.sh/users/5179764), ::{ flag=CN }:: [AlexDunk](https://osu.ppy.sh/users/9194799), ::{ flag=PH }:: [buenoo](https://osu.ppy.sh/users/11799958), ::{ flag=MY }:: [Iyouka](https://osu.ppy.sh/users/7138499), ::{ flag=MY }:: [Jerry](https://osu.ppy.sh/users/605973), ::{ flag=VN }:: [KitHoly](https://osu.ppy.sh/users/14083855), ::{ flag=ID }:: [LenLitchu](https://osu.ppy.sh/users/34098325), ::{ flag=MY }:: [lous](https://osu.ppy.sh/users/6231292), ::{ flag=MY }:: [mochasan\_](https://osu.ppy.sh/users/23804364), ::{ flag=US }:: [mohca](https://osu.ppy.sh/users/14499702), ::{ flag=MY }:: [PokeF](https://osu.ppy.sh/users/18186563), ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061), ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515), ::{ flag=CN }:: [RiceSS](https://osu.ppy.sh/users/8271436), ::{ flag=SG }:: [TheFunk](https://osu.ppy.sh/users/13981991), ::{ flag=MY }:: [Z419](https://osu.ppy.sh/users/9912966) |
+| Statistician | ::{ flag=VN }:: [BCraftMG](https://osu.ppy.sh/users/13456818), ::{ flag=FR }:: [PatouZ](https://osu.ppy.sh/users/7306522) |
+| Web designer | ::{ flag=VN }:: [longnguyen2004](https://osu.ppy.sh/users/3874031), ::{ flag=VN }:: [TvS SorAKuN](https://osu.ppy.sh/users/11115041) |
+| Video editor | ::{ flag=VN }:: [KitHoly](https://osu.ppy.sh/users/14083855) |
+| Wiki editor | ::{ flag=ID }:: [fajar13k](https://osu.ppy.sh/users/7100002) |
+
+## Links
```

Comment:
Perhaps just a me thing, but having "ASC" in all of the links seems a bit redundant considering we're in the wiki page for it. It should already be implied that these links are related to the tournament in some way, shape, or form.

Would say the only one that makes sense to keep would be the trailer. But the Website, X, and YouTube links can probably have their affixes removed.